basic/exercise1/script.js

```diff
@@ -1,30 +1,22 @@
 'use strict';
 function formatDate(date) {
-
+   
     var days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
     var weekDays = `${days[date.getDay()]}`;
-    var hours = date.getHours();
+    var hours = insertingZero(date.getHours());
     var ampm = hours >= 12 ? 'PM' : 'AM';
     hours = hours % 12;
-    if (hours < 10) {
-        hours = "0" + hours;
-    };
-
-    var minutes = date.getMinutes();
-    if (minutes < 10) {
-        minutes = "0" + minutes;
-    };
-
-    var seconds = date.getSeconds();
-    if (seconds < 10) {
-        seconds = "0" + seconds;
-    };
 
+    var minutes = insertingZero(date.getMinutes());
+    var seconds = insertingZero(date.getSeconds());
     var time = `${weekDays} ${hours} ${ampm} ${minutes}: ${seconds}`;
     return time;
-
 };
 
+ function insertingZero(timeUnit) {
```

Comment:
I wrote you in previous code review to not use continous form here - insertingZero is not a good name here, because:
- inserting means that it does it constantly, whereas it does it once
- it does not insert zero always. It does it only in special circumstances
So name of the function is not adequate to what it does.
It is better to name it as I suggested in previous code review which you ignored - formatAsTwoDigits